rasa/shared/importers/default_config.yml

```diff
@@ -14,12 +14,10 @@ pipeline:
   - name: DIETClassifier
     epochs: 100
     constrain_similarities: true
-    model_confidence: cosine
```

Comment:
should this be
```
model_confidence: linear_norm
```
?

Reply:
We haven't kept `model_confidence: linear_norm` in auto-config. Based on some feedback we can do it later versions.